proto/metadata.proto

```diff
@@ -43,7 +43,7 @@ message Metadata {
     // Vendor of the device.
     ondatra.Device.Vendor vendor = 1;
     // Hardware model of the device.
```

Comment:
update the comment, too. just "model" -> "models"?

Reply:
done